### PR TITLE
Update Gradle wrapper to 6.2

### PR DESCRIPTION
**Pull request by nre-ableton**
This is needed to fix some SSL errors which have been observed on
some machines.

The `gradlew` and `gradlew.bat` scripts are autogenerated by the `gradle` tool.

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```
Comment (dgl-ableton) [inline gradlew]: Could you explain what `-Xmx64m` and `-Xms64m` is?

File: gradlew [hunk 2]
```diff
@@ -138,19 +154,19 @@ if $cygwin ; then
...
+        0) set -- ;;
+        1) set -- "$args0" ;;
+        2) set -- "$args0" "$args1" ;;
+        3) set -- "$args0" "$args1" "$args2" ;;
+        4) set -- "$args0" "$args1" "$args2" "$args3" ;;
+        5) set -- "$args0" "$args1" "$args2" "$args3" "$args4" ;;
+        6) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" ;;
+        7) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" "$args6" ;;
+        8) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" "$args6" "$args7" ;;
+        9) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" "$args6" "$args7" "$args8" ;;
```
Comment (dgl-ableton) [inline gradlew]: Could you please explain what is happening here? 😅

File: gradlew [hunk 3]
```diff
@@ -159,14 +175,9 @@ save () {
     for i do printf %s\\n "$i" | sed "s/'/'\\\\''/g;1s/^/'/;\$s/\$/' \\\\/" ; done
     echo " "
 }
-APP_ARGS=$(save "$@")
+APP_ARGS=`save "$@"`
```
Comment (dgl-ableton) [inline gradlew]: What is `"$@"`?

File: gradlew [hunk 4]
```diff
@@ -159,14 +175,9 @@ save () {
...
 
 # Collect all arguments for the java command, following the shell quoting and substitution rules
 eval set -- $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS "\"-Dorg.gradle.appname=$APP_BASE_NAME\"" -classpath "\"$CLASSPATH\"" org.gradle.wrapper.GradleWrapperMain "$APP_ARGS"
 
-# by default we should be in the correct project dir, but when run from Finder on Mac, the cwd is wrong
```
Comment (dgl-ableton) [inline gradlew]: "when run from Finder" -> "when _running_ from Finder"

File: gradlew.bat [hunk 5]
```diff
@@ -13,8 +29,11 @@ if "%DIRNAME%" == "" set DIRNAME=.
 set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
+@rem Resolve any "." and ".." in APP_HOME to make it shorter.
+for %%i in ("%APP_HOME%") do set APP_HOME=%%~fi
```
Comment (dgl-ableton) [inline gradlew.bat]: What is `%%~fi`?

File: gradlew (hunk 1 above)
Comment (nre-ableton) [reply gradlew]: They are Java options for the stack size. But I wouldn't bother reviewing `gradlew` because it is auto-generated by the `gradle` binary.

**Comment by nre-ableton** [comment]
@dgl-ableton Sorry, I should have mentioned that the wrapper scripts are autogenerated by the `gradle` tool and don't need to be reviewed. I've updated the PR description with this information!

**Comment by nre-ableton** [comment]
@dgl-ableton The other team members have all reviewed and approved similar PRs (this was part of a flood 🌊).

**Pull request merged**